cmd/clusterctl/README.md

```diff
@@ -4,6 +4,19 @@
 
 Read the [experience doc here](https://docs.google.com/document/d/1-sYb3EdkRga49nULH1kSwuQFf1o6GvAw_POrsNo5d8c/edit#). To gain viewing permissions, please join either the [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev) or [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) google group.
 
+## Goals
+
+* Provide a bootstrapping mechanism via `kind` or `minikube`
```

Comment:
This was the original purpose of `clusterctl`. Personally I feel like this should be the only use case and we should have dedicated tools for other use cases/workflows.